lib/msal-common/test/error/InteractionRequiredAuthError.spec.ts

```diff
@@ -7,7 +7,11 @@ describe("InteractionRequiredAuthError.ts Class Unit Tests", () => {
     it("InteractionRequiredAuthError object can be created", () => {
```

Comment:
I don't think this test is enough. 

I recommend an E2E test: 

1. copy and modify an existing test which tests for example AcquireTokenForCredential or any other `AcqurieToken`
2. Inject an HTTP response with http status 400 and the body is the JSON with claims
3. Wait for the exception to be thrown and catch it
4. Assert the exception type and the claims